src/main/java/sirius/web/http/Response.java

```diff
@@ -812,11 +799,17 @@ public void redirectToGet(String url) {
      * @param url the URL to redirect to
      */
     public void redirectPermanently(String url) {
+        performRedirect(url, HttpResponseStatus.MOVED_PERMANENTLY);
+    }
+
+    private void performRedirect(String url, HttpResponseStatus status) {
         if (cacheSeconds == null || cacheSeconds == 0) {
             userMessagesCache.cacheUserMessages(webContext);
         }
```

Comment:
did we challenge this logic with our new setup / approach?

Reply:
Good point, will have an eye on this, but leave it as is for now